h2o-py/tests/testdir_algos/glm/pyunit_PUBDEV_7481_glm_startvals.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator as glm
+
+# test startval to set GLM coefficients
+def set_glm_startvals():
+    # read in the dataset and construct training set (and validation set)
+    d = h2o.import_file(path=pyunit_utils.locate("smalldata/logreg/prostate.csv"))
+    mL = glm(family='binomial')
+    mL.train(training_frame=d,x=[2,3,4,5,6,7,8],y=1)
+    mLcoeff = mL.coef()
+    r = glm.getGLMRegularizationPath(mL)
+    rcoeff = r["coefficients"][0]
+    startval1 = [0,0,0,0,0,0,0,-0.3945120960889672]
```

Comment:
`-0.3945120960889672`? where does this come from?

Reply:
@sebhrusen I cheated here and look into glm.java computeImpl method and check what the initial coefficient values are and put it there.

Reply:
I actually have the test generate the initial intercept values nowl